lms/djangoapps/certificates/admin.py

```diff
@@ -28,6 +29,11 @@ def __init__(self, *args, **kwargs):
         self.fields['organization_id'] = forms.TypedChoiceField(
             choices=org_choices, required=False, coerce=int, empty_value=None
         )
+        lang_choices = list(settings.CERTIFICATE_TEMPLATE_LANGUAGES)
+        lang_choices.insert(0, (None, '------------'))
+        self.fields['language'] = forms.TypedChoiceField(
+            choices=lang_choices, required=False, coerce=str, empty_value=None
```

Comment:
By coercing to `str`, I believe we would save `''` instead of `None` for templates for which we did not select a language. I think it's fine if we want to do that, but we should make sure we're consistent about how we denote an unselected language. If we're going with an empty string, we should default to that so that all existing records get the empty string instead of None.